WORKSPACE

```diff
@@ -11,10 +11,13 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 #    reliable downloads.
 http_archive(
     name = "org_tensorflow",
-    sha256 = "a17951020295c8bbf129787415efdc926128efaff5914b6c66fb1b5c763885a9",
-    strip_prefix = "tensorflow-7a29693084fe074aaea588e0de46479404006146",
+    patch_args = ["-p1"],
+    patch_tool = "patch",
+    patches = ["//third_party/tensorflow_patches:tf_pr_48546.patch"],
```

Comment:
Nice!

Reply:
Yeah it's nice isn't it :)

The only problem is what to do about the makefile build. I discussed with @Tombana whether we even need the makefile build any more and we decided that we probably still do, since it is much easier to use than Bazel on e.g. a Raspberry Pi compiling natively.

But then on the other hand this is just a patch to Bazel build files, so I guess we don't need to apply it for the makefile build.